components/UI/Drawer/components/OtherRecordsInSameLocation.tsx

```diff
@@ -11,14 +12,33 @@ type Props = {
 export const CloseByRecord = ({ drawerData }: Props) => {
   const { setDrawerData } = useMapActions();
   const { t } = useTranslation("home");
+  const router = useRouter();
+
   if (
     !drawerData ||
     !("closeByRecords" in drawerData) ||
     !drawerData.closeByRecords ||
     drawerData.closeByRecords.length <= 1
   )
     return null;
+
+  const updateStateID = (reference: number) => {
+    router.replace(
```

Comment:
this needs to be a `router.push`, this will prevent browser's `back` button to not work

Reply:
But @usirin if we change this and click to back button, selected pin and url will not match. What should we do? 
![image](https://user-images.githubusercontent.com/61375971/218731001-290a6f1e-470e-41a0-8cea-2c49a88e5c0e.png)